### PR TITLE
Ajc run optimus data 517

### PR DESCRIPTION
**Pull request by ambrosejcarr**
Added a notebook chronicling the creation of the new indexes needed to run the 10x v2 data. 
Added a notebook describing how to run each downloaded public dataset through Optimus [WIP, waiting on completion of a few dataset] 

I want to get this reviewed while the last few datasets are running so we can get it incorporated this sprint.

### CODE REVIEW

**Comment by jishuxu** [comment]
Are you going to run Picard on the bam files? and what type of metrics are you going to collect? if you will collect RNAmetrics by using Picard, you may need refFlat and rrna_interval files. you can go to HISAT2 reference bundle to find these two files. they should be generic. but I am sure about the mouse reference.

File: analyses/optimus/process_10x_v2_datasets.ipynb [hunk 1]
```diff
@@ -0,0 +1,652 @@
...
+   "cell_type": "code",
+   "execution_count": 156,
+   "metadata": {
+    "collapsed": false,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "inputs_part2 = inputs[:5] + inputs[6:9] + inputs[11:14]"
```
Comment (rexwangcc) [inline analyses/optimus/process_10x_v2_datasets.ipynb]: Why slicing in this way? `:
inputs_part2 = inputs[:5] + inputs[6:9] + inputs[11:14]`

File: analyses/optimus/process_10x_v2_datasets.ipynb [hunk 2]
```diff
@@ -0,0 +1,652 @@
...
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "# grab the dependencies; these are static\n",
+    "workflow_dependencies = {\n",
+    "    \"StarAlignBamSingleEnd.wdl\": \"../../../skylab/library/tasks/StarAlignBamSingleEnd.wdl\",\n",
```
Comment (kbergin) [inline analyses/optimus/process_10x_v2_datasets.ipynb]: Are all these paths with ../../../ always going to work?

File: analyses/optimus/process_10x_v2_datasets.ipynb (hunk 1 above)
Comment (ambrosejcarr) [reply analyses/optimus/process_10x_v2_datasets.ipynb]: Residual metadata (some were failing and I needed to rerun). Will remove. Thanks for catching.

File: analyses/optimus/process_10x_v2_datasets.ipynb (hunk 2 above)
Comment (ambrosejcarr) [reply analyses/optimus/process_10x_v2_datasets.ipynb]: No. I'll make a variable for the skylab directory. Thanks for the comment.

**Pull request merged**